ci_build.sh

```diff
@@ -4,17 +4,9 @@
 #  THIS FILE IS 100% GENERATED BY ZPROJECT; DO NOT EDIT EXCEPT EXPERIMENTALLY  #
 #  READ THE ZPROJECT/README.MD FOR INFORMATION ABOUT MAKING PERMANENT CHANGES. #
 ################################################################################
-# NOTE: This script was customized (PATH, added commented-away quicktest lines,#
```

Comment:
Project regeneration did not retain customizations (see our doc on proper procedure to mix the two worlds), and seems to use an older version of zproject templates. In short, using `git difftool` is essential to review codebase changes after regenerating.

Reply:
Seems to be fixed w/ zproject upgrade...